### ISSUE TITLE
Upgrade grpc-java library to v1.0.0-pre1 [phab:D994]

### ISSUE THREAD

**Issue by kythe** [google/kythe#1328]
_Created by **zdapeng** at 2016-08-02 18:48:59_: 

- grpc-java library is upgraded to v1.0.0-pre1
- netty library is upgraded to v4.1.3
- The service interfaces and the static bindService method are not generated by the grpc-java proto plug-in.

**Comment by kythe** [comment]
_**zarko** wrote at 2016-08-02 22:23:46_:

This patch does not apply against a fresh Kythe checkout at HEAD:

```
error: the patch applies to 'third_party/netty/netty-all-4.1.0.Beta6.jar' (f69d0eb8130ee2cc75df51d1f91e2b32a0a91297), which does not match the current contents.
error: third_party/netty/netty-all-4.1.0.Beta6.jar: patch does not apply
```

**Comment by kythe** [comment]
_**zdapeng** wrote at 2016-08-02 22:41:35_:

add netty-all-4.1.0.Beta6.jar back

**Comment by kythe** [comment]
_**zarko** wrote at 2016-08-02 22:55:01_:

This worked (provided I remove netty-all-4.1.0.Beta6.jar and add a fresh copy of netty-all-4.1.3.Final.jar; arcanist has trouble with binaries). Looking over the rest now.

**Comment by kythe** [comment]
_**zarko** accepted this revision at 2016-08-02 23:09:06_

**Comment by kythe** [comment]
_**zarko** closed via commit fe7497a8aa9308fada1abf03a64f3aa7cec4bb41 at 2016-08-02 23:09:39_

**Comment by kythe** [comment]
commit 7cc5ed0bed7e5db60ecd4ef76b776a4a47794769
